ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/NewMigrationCommand.java

```diff
@@ -18,20 +18,77 @@
 import com.github.rvesse.airline.annotations.Arguments;
 import com.github.rvesse.airline.annotations.Command;
 import com.github.rvesse.airline.annotations.restrictions.Required;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystemException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Command(
     name = "new",
     description = "Creates a new migrations project, directory structure and config file."
 )
 public class NewMigrationCommand extends BaseCommand {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(NewMigrationCommand.class);
+
   @Required
   @Arguments(description = "the project path to create the directory", title = "project-path")
   private String projectPath;
 
   @Override
   public void run() {
-    throw new UnsupportedOperationException();
+    final long startTime = System.currentTimeMillis();
+
+    if (tryCreateDirectory(projectPath) &&
+        tryCreateDirectory(projectPath + "/migrations") &&
+        tryCreatePropertiesFile(projectPath + "/ksql-migrations.properties")) {
+      final long endTime = System.currentTimeMillis();
+      LOGGER.info("Migrations project directory created successfully (execution time " +
```

Comment:
I am confused. Why is the message sent to the logger and not the command output?

Reply:
For now, the log4j config that we provide defaults to stdout. We might revisit this later.